### ISSUE TITLE
Improve accessibility of links and headings on "Your Actions"

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeCENMigration#1465]
The links that need improving are:

1. Instructional Videos

2. please click here

3. this document [when logged in] 

4. a second 'click here' [when logged in]

There could be other links that I do not see, even when logged in, that could do with re-factoring to make accessible to (say) a manager who needs to use a screen reader.

In addition, there is a typo:
"what you were doing at the time and what was happened."
should be
"what you were doing at the time and what has happened."

Plus, there are headings which should be marked up as such — currently they are simply bold text.

**Comment by AlOneill** [comment]
The fist link should be:
`<a target="_blank" href="/cms/instructional-videos">Instructional Videos (opens in new tab)</a><br>`
i.e. the information that the link opens in a new tab should be part of the link text so that it is listed in full for a screen reader user.

The second link should be something along the lines of `please use this link to <a href="https://groups.google.com/a/freeukgenealogy.org.uk/d/forum/freecen-transcribers" target="_blank">sign up for the transcribers group</a>`. The link text should make the destination clear without the context of the full page.

The typo and its fix are given in the description box.

**Comment by AlOneill** [comment]
I've remembered that I can edit much of the text on "Your Actions" as it is on Refinery as "Information for Members". However, I have a nasty suspicion that any changes that I make will be overwritten by the next (developer?) update to that partial page.

The first link described above is missing from the partial page, so I cannot correct it.

Second link fixed plus some typos.

Now I can see the markup I realise that none of the things that look like headings are marked up as headings, except for "Actions" which is an out-of-hierarchy `h5`. DRAFT created to propose a possible fix for all headings except "Actions", which would have to be edited elsewhere. https://test3.freecen.org.uk/cms/system-message-texts/information-for-members/draft-information-for-members

How best to ensure that markup is corrected and not overwritten?

**Comment by DeniseColbert** [comment]
@Vino-S thinks that the text won't be overwritten. We have plans to shortly updated the Release Notes section further down but this is the only part that will be overwritten.

**Comment by AlOneill** [comment]
The [DRAFT Information for Members](https://test3.freecen.org.uk/cms/system-message-texts/information-for-members/draft-information-for-members) page, which forms the LH column of "Your Action", **has** been overwritten. I've improved the markup again using hierarchical headings, edited for size with `class="delta"`. The published version of Information for Members remains non-semantic (no headings marked up).

I do not want to have to put the headings back in every time a new set of release notes is added — we need a system that preserves the headings and any other changes made to improve accessibility.

I cannot edit "Actions" to make it an actual `<h2>` heading.

Do we want the link to the Transcribers group to continue to open in a new tab?

**Comment by DeniseColbert** [comment]
@Vino-S will take a look at the possibility of preserving headings @AlOneill and change Actions to `<h2>` (also check that this will be preserved).

Please treat the link to the Transcribers' Group as we do others.

**Comment by AlOneill** [comment]
@DeniseColbert  
Have put headings in place of bold text as appropriate. Also set link to open in the same tab. If [Draft Information for members](https://test3.freecen.org.uk/cms/system-message-texts/information-for-members/draft-information-for-members) is acceptable, I'll publish it on test 3, ready for the next sync.

**Comment by AlOneill** [comment]
Some observations on some of the parts that, in the main, I cannot edit because they are controlled by the app and need a developer to edit:
 - there does not appear to be an `h1`, neither visible nor hidden, although judging by the browser tab label, the page `title` element looks to be "Action page", while the main menu link is labelled as "Your Actions". I suggest that all three, the `h1` the `title` element and the main menu link should be the same, perhaps "Your Actions". The `h1` could hidden from view using `class="accessibility"`. However, this is not perfect as there is much more on the page than the member's actions.
 - the markup for the "Actions" heading is curious — `<h5 class="beta  text--teal">Actions</h5>` — i.e. it is a completely out of the blue `h5`, presented the size of an `h2`. I suggest that it should be an `h2` at natural size

How about we head the left column with an `h2` "Information for members"? and then adjust the heading levels below it. "Actions" would remain an `h2`.

Comments? Suggestions?
